cfn-lambda/handler/utils.go

```diff
@@ -0,0 +1,65 @@
+package handler
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/lambda"
+	"github.com/aws/aws-sdk-go/service/lambda/lambdaiface"
+	"github.com/logzio/firehose-logs/common"
+)
+
+const lambdaToTrigger = "-log-group-events-lambda"
+
+type LambdaClient struct {
```

Comment:
What is the advantage of having this struct instead of just using `lambdaiface.LambdaAPI` ?

Reply:
We need it for the mocks in the tests